definitions/infra-awselb/golden_metrics.yml

```diff
@@ -28,6 +30,7 @@ latency:
       eventName: entityName
 errorRate:
   title: Error Rate
+  unit: PERCENTAGE
```

Comment:
Should this be op-req per second?

Reply:
lol, y'all just had me change this to PERCENTAGE yesterday.